frontend/src/components/accountPage/UserInfo/UserInfo.tsx

```diff
@@ -0,0 +1,34 @@
+import { FC } from 'react';
+
+export interface IUserInfoProps {
+  fullName: string;
+  username: string;
+  email: string;
+  studentID: string;
+}
+
+const UserInfo: FC<IUserInfoProps> = props => {
+  const { fullName, username, email, studentID } = props;
+  return (
+    <>
+      <h3>Personal</h3>
+      <p>
+        Full name: <strong>{fullName}</strong>
+      </p>
+      <h3>Login</h3>
+      <p>
+        Username: <strong>{username}</strong>
+        <br />
+        Password: <strong>••••••</strong>
```

Comment:
SInce now we don't give the ability to change the password, I would not show the user this field, maybe we can look into how to trigger keycloak to send the `reset password` email and show the user a button to change the password